### ISSUE TITLE
[intel]crash directly after running 2-3 steps

### ISSUE THREAD

**Issue by cunkai** [comfyanonymous/ComfyUI#3592]
After updating to this git branch, the Intel graphics card will crash directly after running 2-3 steps. Branch: 4ae1515f14d338a347fdf8f647e062f5bc17d196 Slightly faster latent2rgb previews.

got prompt
model_type EPS
Using pytorch attention in VAE
Using pytorch attention in VAE
Requested to load SD1ClipModel
Loading 1 new model
D:\Apps\AnaConda3\envs\ComfyUI\Lib\site-packages\intel_extension_for_pytorch\frontend.py:465: UserWarning: Conv BatchNorm folding failed during the optimize process.
  warnings.warn(
D:\Apps\AnaConda3\envs\ComfyUI\Lib\site-packages\intel_extension_for_pytorch\frontend.py:472: UserWarning: Linear BatchNorm folding failed during the optimize process.
  warnings.warn(
Requested to load BaseModel
Loading 1 new model
 15%|████████████▍                                                                      | 3/20 [00:03<00:15,  1.10it/s]

**Comment by cunkai** [comment]
After replacing

`).to(device="cpu", dtype=torch.uint8, non_blocking=comfy.model_management.device_supports_non_blocking(latent_image.device))
`

with 

`.byte()).cpu()`

it works normally.

**Comment by simonlui** [comment]
@cunkai This was fixed with my commit at https://github.com/comfyanonymous/ComfyUI/commit/5eb98f00927ace00b6b3d01ed9c76b113fc4ec9f, just FYI, sorry I didn't see you had an issue or I would've expedited the patch before forgetting about it.
